commit dab4a6d39940e7e8ade06fc74f313b18e7b52710 (refs/changes/28/5228/2)
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2018-12-11T11:05:55-07:00 (10 months ago)
    
    joyent/node-fast#22 Fix copy-paste bug in fast_server module
    Reviewed by: Kody A Kantor <kody@kkantor.com>

diff --git a/CHANGES.md b/CHANGES.md
index 49f7d4d..2fbab3e 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 None yet.
 
+## v2.6.1
+
+* #22 Fix copy-paste bug in fast_server module
+
 ## v2.6.0
 
 * `fast_client_request_time_ms` became `fast_client_request_time_seconds` and is
diff --git a/lib/fast_server.js b/lib/fast_server.js
index 1c633cb..4cc50c0 100644
--- a/lib/fast_server.js
+++ b/lib/fast_server.js
@@ -227,7 +227,7 @@ function FastServer(args)
 	this.fs_nrequests_failed = 0;		/* count of reqs failed */
 
 	if (this.fs_collector) {
-		if (this.fc_collector.FIXED_BUCKETS === true) {
+		if (this.fs_collector.FIXED_BUCKETS === true) {
 			fixed_buckets = true;
 		}
 		this.fs_request_counter = this.fs_collector.counter({
diff --git a/package.json b/package.json
index c21d0f0..099d849 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "fast",
 	"description": "streaming JSON RPC over TCP",
-	"version": "2.6.0",
+	"version": "2.6.1",
 	"main": "./lib/fast.js",
 	"repository": {
 		"type": "git",
